Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -214,15 +253,19 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
 
   }
 
+  internal func paymentCardTextFieldDidEndEditing(_ textField: STPPaymentCardTextField) {
+    self.viewModel.inputs.paymentCardTextFieldDidEndEditing()
+  }
+
   // MARK: - Private Functions
-  private func createStripeToken(cardholderName: String, cardNumber: String, expirationMonth: Month,
-                                 expirationYear: Year, cvc: String) {
+  private func createStripeToken(paymentDetails: PaymentDetails) {
```

Comment:
can I suggest:

```swift
private func createStripeToken(with paymentDetails: PaymentDetails) { ... }
```
So that the callsite reads:

```swift
self?.createStripeToken(with: paymentDetails)
```